src/Client.php

```diff
@@ -595,7 +595,7 @@ private function validateResponse($token)
         if (empty($token->error) === false) {
             throw new Exception($token->error);
         }
-        if (empty($token->accessToken)) {
+        if (empty($token->access_token)) {
```

Comment:
This is right? I thought it was accessToken

![image](https://user-images.githubusercontent.com/6456425/156266589-6c680500-1833-48ab-ba7a-0c766a2e970c.png)